### PR TITLE
store and retrieve time steps

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[save time step in backend](https://trello.com/c/fXnoUCK3/194-save-time-step-in-backend)

File: projects/wave-core-new/src/lib/backend/backend.model.ts [hunk 1]
```diff
@@ -148,3 +149,16 @@ type ParamTypes = string | number | Array<ParamTypes> | { [key: string]: ParamTy
 interface Params {
     [key: string]: ParamTypes;
 }
+
+export interface TimeStepDict {
+    step: number;
+    granularity: TimeStepGranularityDict;
+}
+
+export type TimeStepGranularityDict = 'Millis' |
```
Comment (michaelmattig) [inline projects/wave-core-new/src/lib/backend/backend.model.ts:158]: why is Millis the only unit that is abbreviated?

**Comment by michaelmattig** [comment]
It might make sense to add the TimeConfig button to the side nav as part of this PR

File: projects/wave-core-new/src/lib/backend/backend.model.ts (hunk 1 above)
Comment (ChristianBeilschmidt) [reply projects/wave-core-new/src/lib/backend/backend.model.ts:158]: I don't know, it happened here:
https://github.com/geo-engine/geoengine/pull/110

**Comment by ChristianBeilschmidt** [comment]
> It might make sense to add the TimeConfig button to the side nav as part of this PR


db08e6a

**Pull request merged**